src/lib/ListMetrics.js

```diff
@@ -191,8 +196,26 @@ export default class ListMetrics {
             const metricResponse = this._getMetricResponse(resource, start,
                 end);
             // last 4 are results of storageUtilized, numberOfObjects,
```

Comment:
Comment needs updating for new storageUtilizedCount and numberOfObjectsCount values.